src/core-plugins/ChatGuard.tsx

```diff
@@ -34,29 +34,29 @@ export default class ChatGuard extends Plugin {
         this.patcher.after(ChatInputGuard.default.prototype, "render", (_, component: any) => {
             const chatInput = component.props.children.find(c => c?.props?.accessibilityLabel != undefined);
             const channelId = chatInput?.props?.channel?.id;
-            if(this.CHANNEL_IDS.includes(channelId.toString())){
+            if (+channelId && this.CHANNEL_IDS.includes(String(channelId))) {
```

Comment:
uhhhh whats the point
isnt this the same thing here
```suggestion
            if (this.CHANNEL_IDS.includes(channelId)) {
```